### ISSUE TITLE
The Element.classifyDevice function miss classifiying rotated devices.

### ISSUE THREAD

**Issue by RosarioPulella** [mdgriffith/elm-ui#90]
I believe that the `classifDevice` function can be inaccurate in providing the `DeviceClass`. With this example code https://ellie-app.com/4DHHRcfwVWDa1, on my phone I get this in portrait mode.
![screenshot_20190204-203743](https://user-images.githubusercontent.com/7164552/52248640-5e727480-28bd-11e9-8007-e52115091e8a.png)
And this in landscape mode.
![screenshot_20190204-203750](https://user-images.githubusercontent.com/7164552/52248676-8cf04f80-28bd-11e9-9827-57425a5013b5.png)
I don't think this is behaving correctly.

**Comment by FranzSkuffka** [comment]
It's hard-coded here:
https://github.com/mdgriffith/elm-ui/blob/master/src/Element.elm#L1542

One may assume a tablet has a specific physical height and width - not the pixel count. The solution may be to incorporate DPI in the calculation.

**Comment by mdgriffith** [comment]
Web pixels, as I understand, do have some notion of DPI, they're not literal pixels.  https://developer.mozilla.org/en-US/docs/Web/CSS/length

Make sure you're setting the viewport meta tag:

https://css-tricks.com/probably-use-initial-scale1/

`<meta name="viewport" content="width=device-width, initial-scale=1">`

I realize I should put that in the docs!

All that being said, if you have suggestions on how to do this better, let me know :D.

**Comment by RosarioPulella** [comment]
So I should explain that what i got the images from was a slightly different project than the ellie app. I actually did have the meta tag in the app that I ran in the pictures.

The issue is that the width can change based on orientation, and the code is classifying the device based on width. So this provides an inconsistent classification of the device. The a device can be classified as a phone in one orientation and as a tablet in the other. In pull request I wrote something that should work. #91 

I could be making some wrong assumptions on about the API so if I have something wrong let me know.
